packages/utils/vite.config.integration.ts

```diff
@@ -21,6 +21,7 @@ export default defineConfig({
     include: ['src/**/*.integration.test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
     globalSetup: ['../../global-setup.ts'],
     setupFiles: [
+      './mocks/setup/cliui.mock.ts',
```

Comment:
should be replaced by `test-setup` mocks

Reply:
replaced